app/views/items/destroy.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Items#destroy</h1>
```

Comment:
you will need this file if you plan on deleting with AJAX. If not, please delete.